worker/uniter/relation/statetracker.go

```diff
@@ -123,6 +126,9 @@ func (r *relationStateTracker) loadInitialState() error {
 		r.relationCreated[rel.Id()] = true
 	}
 
+	if r.logger.IsTraceEnabled() {
+		r.logger.Tracef("initialising relation state tracker: %# v", pretty.Formatter(r.stateMgr.(*stateManager).relationState))
```

Comment:
could this be something other than *stateManager or be untyped nil?

Reply:
No, stateMgr is always set to a concrete value